api/repositories/droplet_repository_test.go

```diff
@@ -140,6 +140,7 @@ var _ = Describe("DropletRepository", func() {
 						Expect(createdAt).To(BeTemporally("~", time.Now(), timeCheckThreshold*time.Second))
 					})
 
+					// TODO: Fix these failing UpdatedAt assertions or remove these tests
```

Comment:
I eventually figured out that this happens when I run the envtest tests without specifying the `KUBEBUILDER_ASSETS` env var. I'll remove the comment.